ports/gstplugin/servowebsrc.rs

```diff
@@ -55,8 +55,10 @@ use gstreamer_base::BaseSrcExt;
 use gstreamer_gl::GLContext;
 use gstreamer_gl::GLContextExt;
 use gstreamer_gl::GLContextExtManual;
+use gstreamer_gl_sys::gst_gl_context_thread_add;
```

Comment:
is there no exported rust function for this? should there be one? we should file an upstream bug

Reply:
Filed https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/issues/229